lib/Parser/CharClassifier.cpp

```diff
@@ -419,11 +419,12 @@ Js::CharClassifier::CharClassifier(ScriptContext * scriptContext)
     }
 #endif
 
-#if ENABLE_UNICODE_API
     // If we're in ES6 mode, and we have full support for Unicode character classification
     // from an external library, then use the ES6/Surrogate pair supported versions of the functions
     // Otherwise, fallback to the ES5 versions which don't need an external library
+#if ENABLE_UNICODE_API
```

Comment:
Curious, why did you chose to use the ES6 functions instead of the default functions if ENABLE_UNICODE_API is not enabled?